web-client/src/main/java/no/nav/familie/webflux/builder/AzureWebClientConfig.kt

```diff
@@ -12,34 +14,49 @@ import org.springframework.web.reactive.function.client.WebClient
 @Suppress("SpringFacetCodeInspection")
 @Configuration
 @Import(
-    ConsumerIdFilterFunction::class,
-    InternLoggerFilterFunction::class,
-    BearerTokenFilterFunction::class,
-    RestTemplateBuilderBean::class,
-    NaisProxyCustomizer::class,
-    NaisNoProxyCustomizer::class
+    WebClientConfig::class,
+    RestTemplateBuilderConfig::class,
```

Comment:
Trenger vi ikke å "importe" InternLoggerFilter?
Ser at ConsumerIdFilter og Mdc... blir brukt i WebClientConfig, men fant ikke bruk av internLoggerFilter :... :/

Reply:
Bra notert! Hadde glømt å navne den, la til dette nå! Tenker vi kan ta opp den på gjenomgangen i morgen
* Fjernet bruk av InternLoggerFilter, den brukes ikke i http-client, og vet ikke om vi har behov for den? 

Vi i ef-sak bruker den ikke, men tror BA bruker den? 
Den har "AuditLogger", men som logger på feil format, og som egentlige ikke er bruk/mappet noe sted(?)

Reply:
👍 Sant, tror det er 2 applikasjoner som har egne interceptors som gjør liknende sak
https://logs.adeo.no/goto/dfcd50a0-4ee7-11ed-8607-d590fd125f80

`ba-dokgen` som har https://github.com/navikt/dokgen/blob/0eacd292dbadf8774e23b6ad1b1d8ebb19261d3a/src/main/kotlin/no/nav/dokgen/configuration/LoggerInterceptor.kt

Og `familie-ba-skatteetaten-api` som har https://github.com/navikt/familie-ba-skatteetaten-api/blob/main/src/main/kotlin/no/nav/familie/ba/skatteetaten/interceptor/MaskinportenTokenLoggingInterceptor.kt

Reply:
skatteetaten-api-interceptoren er for å metrikker på hvem som kaller oss siden den er på nettet, men godkjennes statsbudsjettet i desember så kan vi nok skru av hele appen. Så den kommer vi ikke til å oppgradere til webclient før det